config/200-clusterrole.yaml

```
@@ -100,3 +100,14 @@ rules:
 - apiGroups: ["servicebinding.io"]
   resources: ["servicebindings"]
   verbs: ["get","list","watch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: service-binding-provisioned-service
+  labels:
+    servicebinding.io/controller: "true"
+rules:
+- apiGroups: ["bindings.labs.vmware.com"]
+  resources: ["*"]
```

since the intent here is to give other people access, let me specific about the resource

```suggestion
  resources: ["provisionedservices"]
```